action.yml

```
@@ -17,7 +17,7 @@ outputs:
   base_sha:
     description: 'Report base SHA'
   head_sha:
-    description: 'Report head SHA'
+     description: 'Report head SHA'
```

```suggestion
    description: 'Report head SHA'
```

Would that inserted space break the YAML formatting?